ui/StatusQ/src/typesregistration.cpp

```diff
@@ -134,7 +134,15 @@ void registerStatusQTypes() {
     qmlRegisterModuleImport("Qt.labs.calendar", QQmlModuleImportModuleAny,
                             "QtQuick.Controls", QQmlModuleImportLatest);
 #endif
-        
+
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
+    qmlRegisterModule("Qt.labs.settings", 1, 1);
```

Comment:
Please add explanation why it's needed there, because it's not clear, relying only on docs it shouldn't be needed there.

Reply:
It would look something like // Qt.labs.settings has been deprecated since qt 6.5 and moved to QtCore
But that's what the doc is saying as well
https://doc.qt.io/qt-6/qt-labs-settings-qmlmodule-obsolete.html

Reply:
yeah, I know, we will do the cleanup after dropping qt 5. Please just leave note that it's needed for IOS for some reason.